WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -409,7 +409,7 @@ public void onCreate(Bundle savedInstanceState) {
             mHasUpdatedPosts = savedInstanceState.getBoolean(ReaderConstants.KEY_ALREADY_UPDATED);
             mFirstLoad = savedInstanceState.getBoolean(ReaderConstants.KEY_FIRST_LOAD);
             mSearchTabsPos = savedInstanceState.getInt(ReaderConstants.KEY_ACTIVE_SEARCH_TAB, NO_POSITION);
-            mQuickStartEvent = savedInstanceState.getParcelable(QuickStartEvent.KEY);
+            mViewModel.setQuickStartEvent(savedInstanceState.getParcelable(QuickStartEvent.KEY));
```

Comment:
https://user-images.githubusercontent.com/17463767/164388193-a7d2beab-5cb2-49d0-bf6b-ea6ae7156cc6.mp4

Bug:  It looks like accessing `mViewModel ` here will introduce a null pointer crash when the screen is rotated.

We might want to the get the quickStartEvent to some variable and set it on `onActivityCreated`.

Reply:
Nice catch @AjeshRPai! I didn't realize VM was initiated inside `onActivityCreated`. 😅 Looking at the previous code, I wonder why were maintaining `mQuickStartEvent` at all. We only need to access the quick start event inside `onEvent(final QuickStartEvent event)` which already gets the event. I'm thinking to remove `quickStartEvent` variable from the VM for now unless you have any reason to keep it.

Reply:
Yup, you are right. I think we can remove `mQuickStartEvent`. 😄 👍🏼

Reply:
Removed in 73044de.